exporter/terra/gcs/transfer_job.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Dict
+
+
+@dataclass
+class TransferJob:
+    name: str
+    description: str
+    project_id: str
+    source_bucket: str
+    source_path: str
+    aws_access_key_id: str
+    aws_access_key_secret: str
+    dest_bucket: str
+    dest_path: str
+
+    def to_dict(self) -> Dict:
+        start_date = datetime.now()
+        return {
+            'name': self.name,
+            'description': self.description,
+            'status': 'ENABLED',
+            'projectId': self.project_id,
+            'schedule': {
+                'scheduleStartDate': {
+                    'day': start_date.day,
+                    'month': start_date.month,
+                    'year': start_date.year
+                },
+                'scheduleEndDate': {
+                    'day': start_date.day,
+                    'month': start_date.month,
+                    'year': start_date.year
+                }
+            },
+            'transferSpec': {
```

Comment:
Have you considered using the Google types defined here: https://cloud.google.com/python/docs/reference/storagetransfer/latest/google.cloud.storage_transfer_v1.types.TransferSpec

I do not think it is critical, though.